product/opni/components/RoleBinding.vue

```diff
@@ -31,13 +33,26 @@ export default {
       roleName:  '',
       subjects:  [],
       roles:     [],
+      error:    '',
     };
   },
 
   methods: {
-    async save() {
-      await createRoleBinding(this.name, this.roleName, this.subjects);
+    async save(buttonCallback) {
+      if (this.name === '') {
+        this.$set(this, 'error', 'Name is required');
+        buttonCallback(false);
+
+        return;
+      }
+      if (this.roleName === '') {
+        this.$set(this, 'error', 'Role is required');
+        buttonCallback(false);
 
+        return;
+      }
+      await createRoleBinding(this.name, this.roleName, this.subjects);
```

Comment:
We probably also want to add an error if the requests fails. just wrap in a try/catch and you add that error as well.